x-pack/plugins/canvas/public/lib/aeroelastic/matrix.ts

```diff
@@ -47,48 +47,16 @@ const NANMATRIX = [
 
 export const ORIGIN = [0, 0, 0, 1] as vector3d;
 
-export const translate = (x: number, y: number, z: number): transformMatrix3d => [
-  1,
-  0,
-  0,
-  0,
-  0,
-  1,
-  0,
-  0,
-  0,
-  0,
-  1,
-  0,
-  x,
-  y,
-  z,
-  1,
-];
+export const translate = (x: number, y: number, z: number): transformMatrix3d =>
```

Comment:
These functions might eventually be converted to taking a vector or an `{x, y, z}` object rather than separate args, though the code clarity and perf impact isn't too clear to me yet. It might be slower even if we used typed arrays eg. `Float64Array`, due to the need for boxing, and the way `Float64Array`s are created[1].

[1]  First, the array is created, filling it with all zeros on browser infosec grounds rather than letting it uninitialized, then actual contents have to be written again, either directly, or worse, from an untyped JS array that was created just for the purpose. This is why I haven't converted (yet?) the TS typed, but JS untyped arrays and vectors (`transformMatrix3d` etc.) into `Float64Array`s.